### PR TITLE
Fix the previous chatBox espresso test's

### PR DESCRIPTION
**Pull request by kunall17**
015d486: Fixes wrong hint shown in the chatBox

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@kunall17, it looks like you've already signed the Dropbox CLA. Thanks!

File: app/src/main/res/layout/main.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
                 android:dropDownWidth="fill_parent"
-                android:hint="@string/stream_txt"
+                android:hint="@string/subject"
```
Comment (kunall17) [inline app/src/main/res/layout/main.xml]: No, it was just a small fixup so I posted this commit here!

Comment (niftynei) [inline app/src/main/res/layout/main.xml]: Ah ok, thanks!

**Comment by niftynei** [comment]
These compile & run but none of them pass. I think the problem is that I'm logged in when they start. Can you write up some docs on what state your app should be in when these start?  

Functional tests are flakey, I'm a bigger proponent of unit tests for Android stuff.

**Comment by niftynei** [comment]
This has been merged!

**Pull request closed without merging**